client/app/dedicatedCloud/dedicatedCloud.constant.js

```diff
@@ -1,3 +1,14 @@
-angular.module('App').constant('DEDICATED_CLOUD_CONSTANTS', {
-  securityOptions: ['pcidss', 'hds', 'hipaa'],
-});
+angular.module('App')
+  .constant('DEDICATED_CLOUD_CONSTANTS', {
+    securityOptions: ['pcidss', 'hds', 'hipaa'],
+  })
+  .constant('VM_ENCRYPTION_KMS', {
+    regex: {
+      ip: /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/,
```

Comment:
This is terrible, we must have a standardized way to validate data 

(not a critic of your work)